dev-tools/packer/readme.md.j2

```diff
@@ -0,0 +1,23 @@
+# Welcome to {{.beat_name}} {{.version}}
+
+{{.beat_description}}
+
+## Getting Started
+
+To get running with your beat, setup elasticsearch on your localhost and start it with:
```

Comment:
Capital "Elasticsearch". Also the way it's written know it sounds a bit like that is the command to start ES.